report/src/system_evaluation/results.tex

```diff
@@ -1,2 +1,70 @@
-\section{Results}\label{sec:experiment_results}
-%describe in terms of criteria for using bluetooth. `'
\ No newline at end of file
+\section{System evaluation}\label{sec:experiment_results}
+%describe in terms of criteria for using bluetooth. `'
+Having created a map that can be used to classify received measurements, we can now begin to examine wether these classifications are accurate. 
+The measurements are collected using the implemented functionality described in Section \ref{sec:knn_implementation}.
+However, while the room was unoccupied during the creation of the map, this was not the case for this part of the experiment. 
```

Comment:
Maybe we should detail why it did not matter that the room had people in it during the testing. 

Additionally, unoccupied/occupied may not be the right words for this, since we are implying that we are making tests in a room that other people have "booked".